doc: Update docs with Cellular monitor and Serial Terminal apps (#12395)
nrfconnect/sdk-nrf: samples/net/aws_iot/README.rst

To enable more verbose logging from the AWS IoT library, enable the :kconfig:option:`CONFIG_AWS_IOT_LOG_LEVEL_DBG` option.

> * If you are having issues with connectivity on nRF91 Series devices, see the `Cellular Monitor`_ documentation to learn how to capture modem traces in order to debug network traffic in Wireshark.

Reviewer: ```suggestion
* If you have issues with connectivity on nRF91 Series devices, see the `Cellular Monitor`_ documentation to learn how to capture modem traces in order to debug network traffic in Wireshark.
```
